engine/src/workers/cron/skills/skills/cron/reactive-triggers.md

```
@@ -0,0 +1,114 @@
+  "config": {
+    "expression":            "0 0 2 * * * *",         // required. 6- or 7-field cron expression: sec min hour dom month dow [year].
+    "condition_function_id": "jobs::is-weekday"       // optional. Engine invokes this with the event; handler runs only on truthy.
+  }
+}
+```
+
+`type`, `function_id`, and `config.expression` are required. `expression` cannot be missing or empty — the worker rejects the registration synchronously when it can't parse it; the error surfaces as the `Err` from `iii.registerTrigger`.
+
+The supported grammar is **6- or 7-field**, ordered:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)